package.json

```diff
@@ -21,10 +21,10 @@
     "babel-loader": "7.1.2",
     "babel-preset-env": "1.6.1",
     "browser-sync": "2.18.13",
-    "cf-atomic-component": "1.5.3",
+    "cf-atomic-component": "1.5.4",
```

Comment:
did you intend to bump these?

Reply:
Yeah. I bumped them to check that it was correctly paying attention to `node_modules`, and figured might as well leave them in here.